### PR TITLE
PRM Automated Test - LPS-174501 | master

### PR DESCRIPTION
**Pull request by MGabriellaS**
[LPS-174501](https://issues.liferay.com/browse/LPS-174501)

### CODE REVIEW

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:stop

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 1]
```diff
@@ -264,6 +264,67 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-174501. Verify that the only user with the right roles can see/change the state manager"
+	@priority = 5
+	test OnlyManagersCanSeeStateManager {
+		task ("Give I have a user with a manager role") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:270]: Hi @MGabriellaS, just a typo here. Could you please replace 'Give' with 'Given'? Thanks!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 2]
```diff
@@ -264,6 +264,67 @@ definition {
...
+				tactic = "Other",
+				targetMarketOptions = "Education,Energy",
+				targetOfLeads = "test",
+				targetRolesOptions = "Administrator",
+				typeOfActivity = "Miscellaneous Marketing");
+
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name");
+		}
+
+		task ("Then the manager use can see and change the MDF status") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:315]: Hi @MGabriellaS, the same here.
First you have to validate the manager can see the status manager. After that, login as a partner user and validate that the state manager isn't visible. You can see the steps [here](https://docs.google.com/document/d/1MwnucjjTHoJRF5fKv2nROpjPJyU0H9a6jPxvWT6VA8k/edit#)

**Comment by AfonsoHSantos** [comment]
Hi @MGabriellaS, could you please update your master and do a rebase in your branch? Thanks!

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
Hi @MGabriellaS, could you please rename [this commit](https://github.com/pt-liferay-solutions/liferay-portal/pull/3254/commits/078f60ae2a84c5bde674ad1cdb6d6eef816e6899) to match with the commit content

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 3]
```diff
@@ -450,6 +450,88 @@ definition {
...
+		task ("And go to MDF details with a user that does not have a manager role") {
+			PRMUtils.gotoPRM();
+
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "mdf",
+				buttonText = "MDF");
+
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name");
+		}
+		task ("Then the user can't see manager status") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:429]: Hi @MGabriellaS, this task should be 'Then the State Manager isn't visible'

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 4]
```diff
@@ -450,6 +450,88 @@ definition {
...
+			
+			PRMMDFRequest.CanNotViewStateManager();
+		}
+
+		task ("When the manage user go to see MDF details") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:433]: Hi @MGabriellaS, this task should be 'When the Manager goes to the MDF Detail page'

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 5]
```diff
@@ -450,6 +450,88 @@ definition {
...
+
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "mdf",
+				buttonText = "MDF");
+
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name");
+
+		}
+
+		task ("Then the manager use can see and change the MDF status") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:447]: Hi @MGabriellaS, this task should be 'Then the State Manager is visible'

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by MGabriellaS** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
giphy

**Comment by AfonsoHSantos** [comment]
Hi @MGabriellaS, thanks for working on it! The code looks good to me.

**Comment by AfonsoHSantos** [comment]
ci:forward

**Pull request closed without merging**